GW2EIEvtcParser/EIData/ProfHelpers/Guardian/GuardianHelper.cs

```diff
@@ -13,20 +16,43 @@ internal static class GuardianHelper
     {
         internal static readonly List<InstantCastFinder> InstantCastFinder = new List<InstantCastFinder>()
         {
-            new BuffGainCastFinder(ShieldOfWrathSkill, ShieldOfWrathEffect), // Shield of Wrath
-            new BuffGainCastFinder(ZealotsFlameSkill, ZealotsFlameEffect).UsingICD(0), // Zealot's Flame
-            new BuffGainCastFinder(MercifulInterventionSkill, MercifulInterventionSelfEffect),
+            new BuffGainCastFinder(ShieldOfWrathSkill, ShieldOfWrathEffect),
+            new BuffGainCastFinder(ZealotsFlameSkill, ZealotsFlameEffect).UsingICD(0),
             //new BuffLossCastFinder(9115,9114,InstantCastFinder.DefaultICD), // Virtue of Justice
             //new BuffLossCastFinder(9120,9119,InstantCastFinder.DefaultICD), // Virtue of Resolve
             //new BuffLossCastFinder(9118,9113,InstantCastFinder.DefaultICD), // Virtue of Courage
-            new DamageCastFinder(JudgesIntervention,JudgesIntervention), // Judge's Intervention
-            new DamageCastFinder(WrathOfJustice,WrathOfJustice), // Wrath of Justice
-            new DamageCastFinder(SmitersBoon,SmitersBoon), // Smiter's Boon
-            new DamageCastFinder(SmiteCondition,SmiteCondition), // Smite Condition
+
+            // Meditations
+            new BuffGainCastFinder(MercifulInterventionSkill, MercifulInterventionSelfEffect),
+            new DamageCastFinder(JudgesIntervention, JudgesIntervention).UsingDisableWithEffectData(),
+            new EffectCastFinderByDst(JudgesIntervention, EffectGUIDs.GuardianJudgesIntervention).UsingDstBaseSpecChecker(Spec.Guardian),
+            new EffectCastFinderByDst(ContemplationOfPurity, EffectGUIDs.GuardianContemplationOfPurity1).UsingDstBaseSpecChecker(Spec.Guardian),
+            new DamageCastFinder(SmiteCondition, SmiteCondition),
+            new DamageCastFinder(LesserSmiteCondition, LesserSmiteCondition),
+            
+            // Shouts
+            new EffectCastFinderByDst(SaveYourselves, EffectGUIDs.GuardianSaveYourselves).UsingDstBaseSpecChecker(Spec.Guardian),
+            // distinguish by boons, check duration/stacks to counteract pure of voice
+            new EffectCastFinderByDst(Advance, EffectGUIDs.GuardianShout)
```

Comment:
This should be specifically flagged as NotAccurate, in case effect events are not NotAccurate by default in the futur.